x-pack/plugins/lists/common/schemas/types/entries.ts

```diff
@@ -34,9 +34,9 @@ export type EntryMatchAny = t.TypeOf<typeof entriesMatchAny>;
 export const entriesList = t.exact(
   t.type({
     field: t.string,
+    list: t.exact(t.type({ id: t.string, type })),
```

Comment:
NOTE: Changed this to `list` from `value`, SOs were screaming when trying to index `value` as `string | string[] | object` so updated.